ateam_kenobi/src/in_play_eval.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright 2024 A Team
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#include "in_play_eval.hpp"
+
+#include <CGAL/squared_distance_2.h>
+#include <cmath>
+#include <limits>
+
+namespace ateam_kenobi
+{
+
+void InPlayEval::Update(World & world)
```

Comment:
Any specific reason we changed code style in these files to use camel case function names vs. lower case ones? (Doesn't matter but eventually, maybe we should pick a single style?)

Reply:
I don't remember, but I think I was matching something else.
Agreed though, I think once upon a time we talked about using the ROS style guides, which would say lower starting camel case. [ref](https://wiki.ros.org/CppStyleGuide#Function_.2F_Methods)